### DISCUSSION TITLE
Implementing Github 0auth

### DISCUSSION THREAD

**Discussion by Code-Victor** [FirebaseExtended/reactfire#513] [Q&A]
how can I implement github auth with reactfire

**Comment by jhuleatt** [comment]
Hi @Code-Victor, with ReactFire, you can access the `auth` object required by `signInWithPopup` by calling the `useAuth` hook. Once you have that, you can follow the normal docs: https://firebase.google.com/docs/auth/web/github-auth#handle_the_sign-in_flow_with_the_firebase_sdk

**Comment by Code-Victor** [comment]
@jhuleatt oh thanks, it works perfectly.👍
